doc/sections/commands.md

```diff
@@ -301,6 +301,23 @@ whether the defined symbol is non-associative, associative to the right,
 or associative to the left. The priority levels are floating point numbers,
 hence a priority can (almost) always be inserted between two different levels.
 
+**Warning:** No check is done on the syntax of the symbol that is
```

Comment:
This warning paragraph should not be here anymore (it seems). This is probably a bad merge.

Reply:
Deleted in b8fb9af3a00c2fbb135c1dbd2e38fb315051079c